Dockerfile

```diff
@@ -15,18 +17,21 @@ RUN apt-get update -y && \
                        postgresql-client \
                        ruby2.5 ruby2.5-dev \
                        tzdata \
-                       unattended-upgrades \
-                       update-notifier-common \
                        # needed to build some gem native extensions:
                        libz-dev \
     && rm -rf /var/lib/apt/lists/*
 
-RUN gem install -N -v 1.16.1 bundler
-
-RUN mkdir -p /opt/conjur-server
+RUN gem install -N -v 1.17.3 bundler
 
 WORKDIR /opt/conjur-server
 
+# Ensure few required GID0-owned folders to run as a random UID (OpenShift requirement)
```

Comment:
Why move this here? We still have to build the Dockerfile in container-dap to insert the custom `run-conjur` entrypoint right? Isn't it cleaner to have OpenShift-specific functionality added in another repo?

Reply:
This container ("Conjur OSS") should be capable of running in OC without any changes if we want wide adoption and we make that happen with the changes in the container-dap repo. By keeping the changes there, we are not upstreaming our fixes to the proper repo which is kind of backwards of what we should be doing so I moved some of the more critical ones here (though a few more might be needed for running OSS as a limited user in k8s but that was out of this scope).